### ISSUE TITLE
display font size

### ISSUE THREAD

**Issue by ng0177** [lokedhs/maxima-client#4]
The max value of 24 is on a 4k monitor still tiny. Please increase as much as possible. Thanks!

**Comment by lokedhs** [comment]
A better solution would be to detect the DPI and adjust the font sizes appropriately. Did you configure your system to use 2x scaling?

Can you tell me how you run the program? Are you using the precompiled binaries or compiling from source?

**Comment by ng0177** [comment]
![image](https://user-images.githubusercontent.com/34538954/69006911-886dc280-0936-11ea-8787-ec1f47fa1d23.png)

I installed it via Software under Fedora Silverblue. The scale is set to 2x.

**Comment by lokedhs** [comment]
OK, I think I understand what happens now. To confirm, can you show me a screenshot of the application with a single equation displayed?

Also, you can manually set any font size you want by typing `font_size: nn` at the prompt. Where `nn` is the size you want.

**Comment by ng0177** [comment]
Yes, it works but still not satisfactory due to the tiny input size.
![image](https://user-images.githubusercontent.com/34538954/69900501-79463600-1374-11ea-85e4-cc865f5f0310.png)

**Comment by lokedhs** [comment]
Thank you for the update. Can you please show me the output of the following command?

```
xrdb -query | grep dpi
```

Also, can you tell me what the physical dimension of the screen and the resolution you're using? It seems to me that the text has actually adjusted to accommodate a 4k screen (based on the system DPI setting), but if your screen is very small it may still look wrong.

Also, did you set the screen scale to 2x or more?

**Comment by ng0177** [comment]
Xft.dpi:	192

Yes, scale ist set to 2. Otherwise, nothing special - see also on upper screenshort.

**Comment by lokedhs** [comment]
Thank you. It looks like all the scaling is correct, but clearly you expect it to be larger.

Since screenshots don't give a representation of what the output looks like on your particular screen, could you create a new screenshot that also includes content from some other applications? That way I'll be able to see how the scale of Climaxima is different.

Also, can you tell me what the dimensions of your monitor is? Is this  a4k laptop screen?

**Comment by ng0177** [comment]
It is a 27" Monitor from a desktop computer.

![image](https://user-images.githubusercontent.com/34538954/69914268-803b7a00-1442-11ea-82c8-e756acb5d30f.png)

**Comment by lokedhs** [comment]
Thank you very much for the screenshot. Climaxima attempts to scale its text based on the system scale, and as far as I can see, it's only slightly smaller than your desktop text size (compare the size used on the Climaxima menu bar and the text inside the tab in the browser).

However, the actual browser content is much larger. I'm assuming this is because your browser have been configured to display web pages with large font size?

*Question:* Would it be acceptable if I modified Climaxima such that when changing font size the prompt also changes? In other words, the font size would affect everything inside the work area, but not the text in the menu bar itself.

**Comment by ng0177** [comment]
Much appreciate your detailed looking into the problem. The answer to your question is "yes".

**Comment by lokedhs** [comment]
I've been working on implementing this. However, changing the default font size in an application pane is a bit tricky, so instead all text output to the interactor pane needs to be changed. I'd rather not do that, so I'm exploring some other ways of achieving this.

**Comment by ng0177** [comment]
Any progress by chance? We are back to where we left off in Fedora Silverblue 31. I am still having trouble on my 4k Monitor to see that tiny input text. `font_size: nn` has not effect on it.
